### ISSUE TITLE
loss: nan - acc: nan - val_loss: nan - val_acc: nan   Why

### ISSUE THREAD

**Issue by qq191513** [divamgupta/image-segmentation-keras#50]
i follow what the tutorial,but the result is this:
Model output shape (None, 51200, 10)
Epoch 1/1
512/512 [==============================] - 127s 247ms/step - loss: nan - acc: nan - val_loss: nan - val_acc: nan

**Comment by divamgupta** [comment]
Could you share the data for which you see this? Maybe the dataset is very imbalanced.
